Source/JavaScriptCore/llint/WebAssembly64.asm

```diff
@@ -386,7 +465,12 @@ wasmOp(i64_rem_u, WasmI64RemU, macro (ctx)
     else
         error
     end
-    returnq(ctx, t2)
+
+    if X86_64_WIN
+        returnq(ctx, t1)
```

Comment:
can we just give t1/t2 a meaningful name? And maybe above, we should be using the hardware names so that this won't break if we change the names in the future? If this isn't possible that's fine

Reply:
Hmm, I don't know exactly how, but I feel like this can be better. A bunch of these cases seem like they would be redundant with a good local register name plus some real hardware register names

Reply:
Yeah I can give that a try, should be able to pull the register definition to a const and then re-use the same implementation for these division / remainder ops.

Reply:
Gave this a shot, pushed in a second commit. It does reduce some duplication, though I'm not 100% sure whether to extend the variables into the ARM / RISCV branches.

In LowLevelInterpreter we `emit` direct assembly, but to do that you have to emit in Intel syntax for Windows and AT&T for everything else. It's a bit of a shame - that Intel vs. AT&T syntax also introduces a bunch of duplication in offlineasm, which could get eliminated if we used Intel syntax on the non-Windows platform linker (or figure out how to switch from MASM to clang and enable AT&T syntax on Windows).